lib/index.js

```diff
@@ -153,13 +159,14 @@ module.exports = class PrometheusMetricsConsumer extends Writable {
     _write(metric, enc, cb) {
         if (this.typeFor(metric) === metricTypes.HISTOGRAM) {
             if (!isHistogram(metric)) {
-                return cb(
+                cb(
                     new Error(
```

Comment:
Won't passing on an Error here terminate the stream?

Reply:
only if not handled?

Reply:
Yeah. Thing is if its not handeled, the stream will close and then one will be starting to loose metrics. It could take just one bad metric object for such. 

I think its very important to highlight that one need to listen on `error` events for this not to happen. Or; we make sure the module never emit errors...

Reply:
yea good point, which approach do you prefer?
its probably best to just log the error and move on eh?

Reply:
Lets document it. One quickly learn if one start loosing metrics.

Reply:
I started down the road of adding abslog and logging the error and letting the stream continue.

Shall I revert?